Import GTK in a backward compatible way

Landmaj opened the pull request:
This fixes issue #11 without breaking backward compatibility for older versions of MATE. The program will check MATE version and import appropriate GTK. Why is this important? Ubuntu Mate 16.04 is supported until 2021 and that version still requires older version of GTK. A lot of people are stuck at that version (including me) for example at work.

I took the liberty of fixing `setup.py` a little bit, since I had to add a new file there anyway. It used tabs instead of spaces (which are used in all other files - mixed indentation is not allowed in Python 3), there were unnecessary semicolons. I also changed maintainer to author (it makes more sense in your case) and bumped version to 2.0.1.

city41 commented on mate_version.py:
```diff
@@ -0,0 +1,44 @@
+    import gi
+
+    version = get_mate_version()
+    if version and version.major <= 2 and version.minor < 18:
```
should this be `version.major < 2`?

landmaj replied:
That's right, I fixed it.

city41 commented:
Thanks @Landmaj!

landmaj commented:
I removed by accident `gi.require_version("MatePanelApplet", "4.0")`. It's not really required but will throw a warning, so I added it back. 

Bumped version to 2.1.0, as you asked.

Also updated README to reflect the changes, but I kept notice about not addressing GTK2 issues.

landmaj commented:
Tested on Ubuntu Mate 18.04 and 16.04, works without issues.
